app/main/views/marketplace.py

```diff
@@ -111,7 +111,7 @@ def external_404():
     in the page are absolute.
     :return: Our usual 404 page, but with all relative links made absolute
```

Comment:
Can we move this functionality in to dmutils?

Reply:
Do you mean the link patching, or the public 404 handling in general? The former, absolutely.